crates/pop-parachains/src/build.rs

```diff
@@ -1,13 +1,498 @@
 // SPDX-License-Identifier: GPL-3.0
 
+use crate::Error;
+use anyhow::Result;
 use duct::cmd;
-use std::path::PathBuf;
+use pop_common::replace_in_file;
+use serde_json::Value;
+use std::{
+	collections::HashMap,
+	fs,
+	path::{Path, PathBuf},
+};
+use toml_edit::DocumentMut;
 
-/// Build the parachain located in the specified `path`.
-pub fn build_parachain(path: &Option<PathBuf>) -> anyhow::Result<()> {
+/// Build the parachain.
+///
+/// # Arguments
+/// * `path` - Location of the parachain project.
+pub fn build_parachain(path: &Option<PathBuf>) -> Result<(), Error> {
```

Comment:
As suggested elsewhere, from a library perspective this should take `release: bool` as a parameter